src/main/java/org/logstash/dissect/SkipField.java

```diff
@@ -0,0 +1,33 @@
+package org.logstash.dissect;
+
+import com.logstash.Event;
+
+import java.util.Map;
+
+class SkipField extends NormalField {
```

Comment:
I think the class hierarchy is a bit odd if I recall the old rule: <sub class> is a <super class>. In this case "A SkipField is a NormalField"? If you need to share code, I'd rather create an abstract super class and let `NormalField` and `SkipField` be siblings.